tests/apollo/test_skvbc_checkpoints.py

```diff
@@ -474,6 +425,10 @@ async def test_checkpoint_propagation_after_corrupting_checkpoint_msg_for_primar
         """
         bft_network.start_all_replicas()
         skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+        await trio.sleep(2)
```

Comment:
I think ypu should explain here and in line 450 why are you doing this.
It would be very hard for a reader to understand in a short time why we sleep here and why the number of mismatches is expected tp be 3 or 4.
same for line 504.

In general, just think you will need to debug it yourself in 1 year - any chance you will remember yourself the reason you did that?